sendHttpPostRequest works different for OH2

Last week I migrated from OH1 to OH2 and experienced that some of my rules where not working anymore. My rules is about to send a new value to my Volkszaehler for logging purposes. 
The rule which worked within OH1 was:
```
rule "AussenTemperatur_an_Volkszaehler" 
when Item AussenTemperatur changed
then
var String request = ''
request = 'http://vz.server.home/middleware.php/data/13c3cda0.json?value=' + AussenTemperatur.state.toString
logDebug("New value for AussenTemperatur {}",  request);
sendHttpPostRequest(request)
```

To get it working I needed to change the `sendHttpPostRequest(request)` to `sendHttpPostRequest(request, "application/json", "")`.

Within log I could see: 
`20:12:29.860 [INFO ] [marthome.event.ItemStateChangedEvent] - AussenTemperatur changed from 0.3 to 0.4`
`20:12:29.870 [ERROR] [lipse.smarthome.io.net.http.HttpUtil] - Fatal transport error: java.lang.NullPointerException`

Maybe someone can check why the post is not working as within OH1.

--- a/bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java
+++ b/bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java
@@ -207,7 +207,7 @@ public class HttpUtil {
         }
 
         // add content if a valid method is given ...
-        if (method.equals(HttpMethod.POST) || method.equals(HttpMethod.PUT) && content != null) {
+        if (content != null && (method.equals(HttpMethod.POST) || method.equals(HttpMethod.PUT))) {
             request.content(new InputStreamContentProvider(content), contentType);
         }
 

